frontend/playground-frontend/src/components/Feedback/Feedback.js

```diff
@@ -36,11 +37,23 @@ const Feedback = () => {
         </div>
 
         <div className="sections" style={styles.content_section}>
-          <p>Feedback submitted!</p>
+          <p style={{color: "0,0,0", fontSize: "5vh", textAlign: "center"}}>Feedback submitted!</p>
+        </div>
+        <div className="sections" style={styles.content_section}>
+          <p style={{color: "0,0,0", fontSize: "4vh", textAlign: "center"}} >
```

Comment:
make sure to run a `git pull origin dev` to pull latest changes of `dev` branch into this branch @aryemans